src/dmd/backend/cg87.d

```diff
@@ -1857,26 +1899,29 @@ void load87(CodeBuilder& cdb,elem *e,unsigned eoffset,regm_t *pretregs,elem *ele
             default:
             Ldefault:
                 retregs = mST0;
-#if 1           /* Do this instead of codelem() to avoid the freenode(e).
+static if (1)   /* Do this instead of codelem() to avoid the freenode(e).
                    We also lose CSE capability  */
-                if (e->Eoper == OPconst)
+{
+                if (e.Eoper == OPconst)
                 {
-                    load87(cdb, e, 0, &retregs, NULL, -1);
+                    load87(cdb, e, 0, &retregs, null, -1);
                 }
                 else
-                    (*cdxxx[e->Eoper])(cdb,e,&retregs);
-#else
-                codelem(cdb,e,&retregs,FALSE);
-#endif
+                    callcdxxx(cdb,e,&retregs,e.Eoper);
```

Comment:
I presume this was required due to the Windows failures.

Reply:
cdxxx[] is a problematic declaration, and I didn't want to fiddle with it here.